questionnaire/form-helper.js

```diff
@@ -181,7 +181,15 @@ function processPreviousAnswers(answersObject) {
     return answers;
 }
 
-function getSectionHtml(sectionData, allAnswers, csrfToken) {
+function escapeSchemaContent(schema) {
+    // Double escape any "\\" to work around this issue: https://github.com/mozilla/nunjucks/issues/625
+    const schemaAsJson = JSON.stringify(schema);
+    const schemaWithEscapedContent = JSON.parse(schemaAsJson.replace(/\\\\/g, '\\\\\\\\'));
+
+    return schemaWithEscapedContent;
+}
+
+function getSectionHtml(sectionData, allAnswers) {
```

Comment:
missing `csrfToken` function parameter. This is needed for the `renderSection` function call within this function.

Reply:
That's it in now 👍